ibisworld/test/ibisworld.2020-10-15.csv

```diff
@@ -0,0 +1,7 @@
+out-rtype;out-mime;out-platform_name;out-login;out-unitid;in-url
```

Comment:
The field `platform_name` is filled with the value of `manifest.longname` afterward, and thus does not require to be tested here.